install/collabora-install.sh

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env bash
+
+# Copyright (c) 2021-2024 tteck
+# Author: tteck (tteckster)
+# Co-Author: remz1337
+# License: MIT
+# https://github.com/tteck/Proxmox/raw/main/LICENSE
+
+source /dev/stdin <<< "$FUNCTIONS_FILE_PATH"
+color
+verb_ip6
+catch_errors
+setting_up_container
+network_check
+update_os
+
+msg_info "Installing Dependencies"
+$STD apt-get install -y curl
+$STD apt-get install -y sudo
+$STD apt-get install -y mc
+msg_ok "Installed Dependencies"
+
+msg_info "Installing Collabora Online"
+cd /usr/share/keyrings
+$STD wget -q https://collaboraoffice.com/downloads/gpg/collaboraonline-release-keyring.gpg
+cd ~
+cat <<EOF >/etc/apt/sources.list.d/collaboraonline.sources
+Types: deb
+URIs: https://www.collaboraoffice.com/repos/CollaboraOnline/CODE-deb
+Suites: ./
+Signed-By: /usr/share/keyrings/collaboraonline-release-keyring.gpg
+EOF
+$STD apt update 
+$STD apt install -y coolwsd code-brand collaboraoffice*
```

Comment:
According to the [installation procedure](https://www.collaboraonline.com/code/), yours differ from `apt install coolwsd code-brand`, why install `collaboraoffice*`?

Reply:
This is the minimal installation, without localizations. For full installation install ‘collaboraoffice*’ packages.